.circleci/config.yml

```diff
@@ -137,15 +141,21 @@ jobs:
   test-browser:
     <<: *test-defaults
     environment:
-      # browser.test_sdl_ttf_render_text_solid depends on Arial from Microsoft
       # browser.test_sdl2_mouse and/or SDL2 should be fixed. The case happens
       # to be failing here, and the root cause might be related with the
       # initial position of the mouse pointer relative to the canvas.
       # browser.test_html5_webgl_create_context is skipped because
       # anti-aliasing is not well supported.
-      - TEST_TARGET=browser skip:browser.test_sdl_ttf_render_text_solid skip:browser.test_sdl2_mouse skip:browser.test_html5_webgl_create_context
+      # browser.test_webgl_offscreen_canvas_in_pthread and
+      # browser.test_webgl_offscreen_canvas_in_mainthread_after_pthread
+      # are crashing Firefox (bugzil.la/1281796). The former case even
+      # reveals more problems: 1) #4412 made it possible to keep threads
+      # alive for async functions, but the change got reverted by #5878;
```

Comment:
please open an issue for the #5878 regression - I think maybe we just need to require `NO_EXIT_RUNTIME` be false when offscreen canvas, but we need to see. and please link to the issue you open here.

Reply:
See #6897